run.py

```diff
@@ -1,30 +1,24 @@
-import argparse
 
-from src.add_songs import create_db, add_track
+from config import config
+from src.store_data_in_s3 import  write_to_s3
+import logging
 
-if __name__ == '__main__':
+if __name__=='__main__':
 
-    # Add parsers for both creating a database and adding songs to it
-    parser = argparse.ArgumentParser(description="Create and/or add data to database")
-    subparsers = parser.add_subparsers()
+    # s3 configs
+    bucket_name = config.S3_BUCKET_NAME
```

Comment:
perhaps a better way of including environment vars is for the user to just `export` them via terminal commands. it is usually a bad idea for the user to touch your source code (in this case `config.py`). this is usually a source of bugs,

Reply:
Will move some configurations to yaml files.